### ISSUE TITLE
`cargo dev rename_lint` misses the lint pass name in `clippy_lints/src/lib.rs`

### ISSUE THREAD

**Issue by ada4a** [rust-lang/rust-clippy#15976]
### Summary

I had this happen when renaming `empty_enum` to `empty_enums` -- both the file and the lint pass struct were renamed (to `empty_enums` and `EmptyEnums`, respectively), but in `clippy_lints/src/lib.rs`, only the module name was updated, but not the struct name:
```rs
store.register_late_pass(|_| Box::new(empty_enums::EmptyEnum));
```

It would be nice if that was updated as well.

### Reproducer

I tried this code:

```shell
git switch --detach 1ecb18a18211c86a2a2f00ef3b0ef4c54f54d0d2
cargo dev rename_lint empty_enum empty_enums
```

I expected to see this happen:
The struct was renamed in `clippy_lints/src/lib.rs`

Instead, this happened:
It wasn't

### Version

```text
rustc 1.92.0-nightly (57ef8d642 2025-10-15)
binary: rustc
commit-hash: 57ef8d642d21965304bde849bab4f389b4353e27
commit-date: 2025-10-15
host: x86_64-unknown-linux-gnu
release: 1.92.0-nightly
LLVM version: 21.1.3
```

**Comment by samueltardieu** [comment]
Cc @Jarcho

**Comment by antoniobraganca23-lgtm** [comment]
git switch --detach 1ecb18a18211c86a2a2f00ef3b0ef4c54f54d0d2
cargo dev rename_lint empty_enum empty_enums
